src/seedsigner/views/seed_views.py

```diff
@@ -1226,6 +1244,249 @@ def run(self):
 
 
 
+"""****************************************************************************
+    BIP-352: Silent Payments
+****************************************************************************"""
+class SeedBIP352SilentPaymentsOptionsView(View):
+    GENERATE_SP_ADDRESS = ButtonOption("Generate SP address")
+    EXPORT_SPEND_PUBKEY = ButtonOption("Export spend pubkey")
+    EXPORT_SCAN_PRIVKEY = ButtonOption("Export scan privkey")
+
+    def __init__(self, seed_num: int):
+        super().__init__()
+        self.seed_num = seed_num
+
+    def run(self):
+        button_data = [
+            self.GENERATE_SP_ADDRESS,
+            self.EXPORT_SPEND_PUBKEY,
+            self.EXPORT_SCAN_PRIVKEY,
+        ]
+
+        selected_menu_num = self.run_screen(
+            ButtonListScreen,
+            title=_("Silent Payments"),
+            button_data=button_data,
+            is_button_text_centered=False,
+            is_bottom_list=True,
+        )
+
+        if selected_menu_num == RET_CODE__BACK_BUTTON:
+            return Destination(SeedOptionsView, view_args={"seed_num": self.seed_num}, clear_history=True)
+        
+        elif button_data[selected_menu_num] == self.GENERATE_SP_ADDRESS:
+            if self.settings.get_value(SettingsConstants.SETTING__BIP352_SILENT_PAYMENTS) == SettingsConstants.BIP352__ENABLED_WITH_LABELS:
+                return Destination(SeedBIP352LabelPromptView, view_args={"seed_num": self.seed_num})
+            else:
+                return Destination(SeedBIP352GeneratePaymentAddressView, view_args={"seed_num": self.seed_num})
+
+        elif button_data[selected_menu_num] == self.EXPORT_SPEND_PUBKEY:
+            return Destination(SeedBIP352ExportSpendPubkeyDetailsView, view_args={"seed_num": self.seed_num})
+
+        elif button_data[selected_menu_num] == self.EXPORT_SCAN_PRIVKEY:
+            return Destination(SeedBIP352ExportScanPrivkeyDetailsView, view_args={"seed_num": self.seed_num})
+
+
+
+class SeedBIP352ExportSpendPubkeyDetailsView(View):
+    def __init__(self, seed_num: int):
+        super().__init__()
+        self.seed_num = seed_num
+        self.seed = self.controller.get_seed(seed_num)
+        self.network = self.settings.get_value(SettingsConstants.SETTING__NETWORK)
+        self.master_fingerprint = self.seed.get_fingerprint(network=self.network)
+        self.spend_pubkey = self.seed.derive_bip352_spend_privkey(network=self.network).to_public().to_string()
+        self.derivation_path = self.seed._build_bip352_path(is_scan_key=False, network=self.network)
+        
+
+    def run(self):
+        selected_option = self.run_screen(
+            BaseKeyDetailsScreen,
+            title=_("Spend Pubkey Details"),
+            key_label=_("Spend Public Key"),
```

Comment:
More places to drop `TRANSLATOR_NOTE` comments. It is repetitive, but the translation strings are presented to the translators in seemingly random order. So we have no guarantees if this is the first time they're ever encountering the terminology while translating or if it's the 20th.